lntest/itest/lnd_test.go

```diff
@@ -1403,10 +1403,21 @@ test:
 	// new Carol+Dave for each test instance as well.
 	for _, carolCommitType := range allTypes {
 		for _, daveCommitType := range allTypes {
+			carolArgs := carolCommitType.Args()
```

Comment:
this temp variable (along with `daveArgs`) isn't used in this new code. perhaps they don't need to move?

Reply:
yep my bad, now fixed!